### PR TITLE
javascript assignment 17 18 19

### PR DESCRIPTION
**Pull request by harshalkotkar**

### CODE REVIEW

File: HARSHAL/javascript/assignment18/assignment18Q2.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            <button onclick="check()">SHOW</button>
+        </fieldset>
+        <script>
+            function check() {
+                var name;
+                var age;
+                var check;
+                name = document.getElementById("name").value;
+                age = document.getElementById("age").value;
+                if (age > 18) {
```
Comment (MKadeer17) [inline HARSHAL/javascript/assignment18/assignment18Q2.html:35]: Use Try catch (Concept) for throwing error for eligible or not.

File: HARSHAL/javascript/assignment18/assignment18Q3.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            <label for="number">ENTER SHOPPING AMOUNT: </label>
+            <input id="shopping" type="number" />
+            <button onclick="check()">CALCULATE</button>
+            <p>TOTAL AMOUNT: <span id="price"></span></p>
+        </fieldset>
+        <script>
+            function check() {
+                var shopping;
+                shopping = document.getElementById("shopping").value;
+                if (shopping > 1000) {
```
Comment (MKadeer17) [inline HARSHAL/javascript/assignment18/assignment18Q3.html:35]: Use Try catch (concept).

**Review by MKadeer17 — APPROVED** [review]

File: HARSHAL/javascript/assignment18/assignment18Q2.html (hunk 1 above)
Comment (Snehal-113) [reply HARSHAL/javascript/assignment18/assignment18Q2.html:35]: This is also ok

File: HARSHAL/javascript/assignment18/assignment18Q3.html (hunk 2 above)
Comment (Snehal-113) [reply HARSHAL/javascript/assignment18/assignment18Q3.html:35]: this is also ok...try catch we should use for handling exceptions .

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**